### PR TITLE
Add ceph-ansible suite

### PR DESCRIPTION
**Pull request by zmc**
Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

**Comment by zmc** [comment]
@yuriw want to take a look at this please?

File: suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+tasks:
```
Comment (vasukulkarni) [inline suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml]: This should be appended to other tasks in tasks list for better readablity just like we have it for other smoke suite

Comment (zmc) [inline suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml]: Not sure what you mean by this?

Comment (vasukulkarni) [inline suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml]: what I am saying is 2-setup folder is not required if you add this task to the workloads itself, also this I think can get read of the folder prefix 0,1,2,3 etc

Comment (zmc) [inline suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml]: Not all suites repeat the same base set of tasks:
https://github.com/ceph/ceph-qa-suite/blob/master/suites/rbd/basic/base/install.yaml

Comment (zmc) [inline suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml]: Agreed that the numbers can go away, though.

**Comment by yuriw** [comment]
Please us `teuthology-describe-tests` to annotate tests

**Comment by yuriw** [comment]
@zmc Looks good to me !

**Pull request merged**